snapshot/snapshots/fake_checkouter.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/scootdev/scoot/snapshot"
 )
 
+// Create a Checkouter that always fails due to invalid path
```

Comment:
The checkouter is a noop checkouter because it does a noop on checkout, it just returns a static path (which happens to be an invalid one). You can also create a noopCheckouter with a valid path.